addons/wowl/static/src/components/navbar/navbar.ts

```diff
@@ -8,12 +8,20 @@ export class NavBar extends Component<{}, OdooEnv> {
   menuRepo = useService("menus");
   state = useState({ showDropdownMenu: false });
 
-  systrayItems = this.env.registries.systray.getAll();
+  systrayItems = this._getSystrayItems();
 
   toggleDropdownMenu() {
     this.state.showDropdownMenu = !this.state.showDropdownMenu;
   }
 
+  _getSystrayItems() {
+    return this.env.registries.systray.getAll().sort((x, y) => {
+      const xSeq = x.sequence === undefined ? 50 : x.sequence;
```

Comment:
As we make use of transpilation, I think we must take advantage of that.
For instance, here you could use the [nullish coalesce operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator):
```suggestion
      const xSeq = x.sequence ?? 50;
```

Reply:
Good point. For the record, here is how it looks like in JS:
`x.sequence !== null && x.sequence !== void 0 ? x : 50`

Reply:
Good to know thx :) I will fix it